packages/devtools_app/lib/src/standalone_ui/vs_code/mock_environment/flutter_panel_mock_editor.dart

```diff
@@ -0,0 +1,143 @@
+// Copyright 2023 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:collection';
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+
+import '../../../shared/split.dart';
+import '../../../shared/theme.dart';
+import '../flutter_panel.dart';
+import 'dart_tooling_mock_api.dart';
+
+/// A simple UI that acts as a stand-in host IDE to simplify the development
+/// workflow when working on embedded tooling.
+///
+/// This UI interacts with [MockDartToolingApi] to allow triggering events that
+/// would normally be fired by the IDE and also shows a log of recent requests.
+class VsCodeFlutterPanelMockEditor extends StatefulWidget {
```

Comment:
I wonder if this should be implemented as a stager app that lives in our test/ directory, that way it doesn't have to get shipped with our production code. We have stager apps for several DevTools screens that can be used for simplified development and testing.

Reply:
Oh that might be better. I'm not very familiar with stager though - I'll take a look at changing it this week. It would definitely be nicer if we could keep development screen out of the main app. Thanks!